utilities/rpp-performancetests/HOST_NEW/Tensor_host_pln1.cpp

```diff
@@ -25,6 +25,43 @@ typedef half Rpp16f;
 #define RPPMAX2(a,b) ((a > b) ? a : b)
 #define RPPMIN2(a,b) ((a < b) ? a : b)
 
+std::string get_interpolation_type(unsigned int val, RpptInterpolationType &interpolationType)
```

Comment:
consider putting this in a common header file instead of redefining in all .cpp files

Reply:
Will add this to the list of changes for our test suite changes. @sampath1117